ext/symengine/symengine.h

```diff
@@ -11,5 +11,6 @@ VALUE c_basic;
 VALUE c_symbol;
 VALUE c_integer;
 VALUE c_rational;
+VALUE c_constants;
```

Comment:
If I make an empty class and assign it to `c_constants` in `symengine.c`, that would be fine?
and set `Klass_of_basic` to the same as well?